### ISSUE TITLE
Additional Python Packages Installed to Wrong Directory 

### ISSUE THREAD

**Issue by ccampbell9** [flathub/org.qgis.qgis#333]
Expected: `flatpak run --devel --command=pip3 org.qgis.qgis install <package> <--user/--system>`  installs packages to the default `site-packages` directory used by the flatpak: 

1. `~/.local/share/flatpak/app/org.qgis.qgis/x86_64/stable/active/files/lib/python3.10/site-packages/`
2. `/var/lib/flatpak/app/org.qgis.qgis/x86_64/stable/active/files/lib/python3.10/site-packages/`

Actual: `flatpak run --devel --command=pip3 org.qgis.qgis install <package> <--user/--system>` installs packages to `~/.local/lib/python3.10/site-packages `

**Comment by ccampbell9** [comment]
Current work around:

1. Install QGIS at the user level
2. Run `flatpak run --user --devel --command=pip3 org.qgis.qgis install <package>`
3. Locate the files that were just downloaded (on Fedora 38 these are in `~/.local/lib/python3.10/site-packages`)
4. Copy the files to the correct location (on Fedora 38: `~/.local/share/flatpak/app/org.qgis.qgis/x86_64/stable/active/files/lib/python3.10/site-packages/`)

**Comment by firefoxlover** [comment]
for me there is no pip at all?

```
flatpak run --command=/bin/bash org.qgis.qgis
pip
```

gives an error. How to resolve the Plugin dependencies???

**Comment by aleixpol** [comment]
> for me there is no pip at all?
> 
> ```
> flatpak run --command=/bin/bash org.qgis.qgis
> pip
> ```
> 
> gives an error. How to resolve the Plugin dependencies???

Note you are missing `--devel`.

**Comment by firefoxlover** [comment]
So the pip modul install command would be?

```
flatpak run --command=/bin/bash --devel org.qgis.qgis
pip install --target=~/.var/app/org.qgis.qgis/data/QGIS/QGIS3/profiles/default/python/plugins <module-name>
```

It always complains about some `/app/share/qgis/python/qgis/utils.py` to be missing, what directory is that really?

**Comment by JMaravalhasSilva** [comment]
Confirmed that this is also completely broken in Ubuntu 22.04 LTS. 

I was looking forward to using the flatpak version as it should make it much easier to manage some specific dependencies for a weird plugin I need without messing with my system interpreter, but in its current state the flatpak is unusable.

Just like [this user in the gis stack exchange](https://gis.stackexchange.com/questions/466842/qgis-flatpak-installation-modulenotfounderror-under-linux-mint) mentions, attempting to install a module results the following error:
```error: runtime/org.kde.Sdk/x86_64/5.15-22.08 not installed```

Just like that user, I ran ```flatpak install org.kde.Sdk``` which then prompts which version I want to install. I choose the aforementioned version and can now install packages, except they are clearly not correctly installed since QGIS complains that it can't find the installed modules.
